### PR TITLE
Create bubble_sort.java

### LINKED ISSUE #1: Sorting: Bubble Sort
Implement Bubble Sort

### PR DESCRIPTION
**Pull request by rohitjmathew**
Fixes #1 and #4

### CODE REVIEW

**Comment by aayush26** [comment]
Since, everyone is active with hacktoberfest, can you add this file inside the sorting folder that I just created. I am ok with multiple implementations. Just put it inside that folder for better organisation.
I will merge it after that.

**Comment by rohitjmathew** [comment]
Sure. GIve me 2 min

**Comment by rohitjmathew** [comment]
@aayush26 you already did it. Hence no need to do it.

**Comment by aayush26** [comment]
You can pick up insertion sort if you want, give me a minute. I will create an issue.

**Comment by aayush26** [comment]
Actually I didn't see your PR and I saw it later.

**Comment by rohitjmathew** [comment]
> You can pick up insertion sort if you want, give me a minute. I will create an issue.

Working on it @aayush26

**Comment by rohitjmathew** [comment]
@aayush26 Added insertion sort

**Comment by aayush26** [comment]
Can you either remove bubble sort from the commit or move it to sorting folder?
Once I squash and merge, bubble sort will be merged too.

File: sorting/InsertionSort.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+public class insertion_sort {
```
Comment (aayush26) [inline sorting/InsertionSort.java]: change it to InsertionSort
Java, classname and file name must be same

**Comment by rohitjmathew** [comment]
@aayush26 its deleted

File: sorting/InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    /**
+     * insertion sort picks up the current element(i) and checks it against its previous element(j = i-1 until j>=0) and place it
+     * takes O(n^2) and inplace algo with no extra space
+     * @param arr
+     * @return
+     */
+
+    public static int[] insertionSort(int[] arr) {
```
Comment (aayush26) [inline sorting/InsertionSort.java]: final int[] arr

immutability is always thread safe.

File: sorting/InsertionSort.java (hunk 1 above)
Comment (rohitjmathew) [reply sorting/InsertionSort.java]: Forgot. WIll change.

**Pull request merged**